### ISSUE TITLE
URL download not working

### ISSUE THREAD

**Issue by Sir-Will** [DoubleDoorDevelopment/CurseModpackDownloader#11]
With the curseforge site changes it no longer downloads the modpacks manifest file.
It says `Download pack from URL: https...` and then nothing else happens

**Comment by Claycorp** [comment]
Did you build the jar from the source? It should work as someone already PR'ed a change for it.

**Comment by Sir-Will** [comment]
Yes I did. It works fine if I manually download the zip and run it but not with a url.
